src/time/BrokenDate.cpp

```diff
@@ -90,3 +122,28 @@ BrokenDate::DaysSince(const BrokenDate &other) const
   const int64_t delta = a - b;
   return int(delta / SECONDS_PER_DAY);
 }
+
+StaticString<20>
+BrokenDate::c_str() {
+  StaticString<20> date_string;
+  date_string.Format(_T("%04d-%02d-%02d"), year, month, day);
+  return date_string;
+}
+
+StaticString<20>
+BrokenDate::Format(StaticString<20> format) {
+
+  StaticString<20> date_string;
+  if (IsPlausible()) {
+    // TODO(August2111): make a really format dependend output
+    if (format == _T("DD.mm.YYYY")) {
+      date_string.Format(_T("%02d.%02.%04d"), day, month, year);
+    } else if (format == _T("YYYY-mm-DD")) {
+      date_string.Format(_T("%04d-%02-%02d"), year, month, day);
```

Comment:
Oh no, hard-coded date formats, inside a low-level library. Please not.

Reply:
The changes on BrokenDate should be a (very bad...) design proposal, I removed this staff completely.
Personally I prefer all things together: No different classes for BrokenDate/Time, RoughTime, the convertions between strings, doubles, integers and date/times on complete other places (Formatter/TimeFormatter)  
But this should be not part of this PR, so I don't any touch to this class here